src/github/middleware.ts

```diff
@@ -119,14 +154,16 @@ export default (
 			return;
 		}
 
-		context.log(`Processing event for ${jiraSubscriptionsCount} jira instances`);
+		context.log(
+			`Processing event for ${jiraSubscriptionsCount} jira instances`
+		);
```

Comment:
For the future, It is better to avoid reformatting files, hence it makes PR review more difficult

Reply:
I agree. Unfortunately, every time I save it autoformats. I have the same linting set up as the rest of the team and am yet to resolve this.